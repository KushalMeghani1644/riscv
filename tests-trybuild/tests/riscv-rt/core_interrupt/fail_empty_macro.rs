#[riscv_rt::core_interrupt]
fn my_interrupt() {}

fn main() {}
