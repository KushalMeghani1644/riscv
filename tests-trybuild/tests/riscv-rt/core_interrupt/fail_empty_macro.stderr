error: `#[core_interrupt]` attribute expects a path to a variant of an enum that implements the riscv_rt :: CoreInterruptNumber trait.
 --> tests/riscv-rt/core_interrupt/fail_empty_macro.rs:1:1
  |
1 | #[riscv_rt::core_interrupt]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `riscv_rt::core_interrupt` (in Nightly builds, run with -Z macro-backtrace for more info)
